planet/scripts/cli.py

```diff
@@ -103,3 +107,37 @@ def init(email, password):
     response = call_and_wrap(clientv1().login, email, password)
     write_planet_json({'key': response['api_key']})
     click.echo('initialized')
+
+@cli.command('quota')
+def quota():
+    '''Print allocation and remaining quota in Sqkm.'''
+    try:
+        with open (os.path.join(os.path.expanduser('~'), '.planet.json')) as f:
+            data=json.load(f)
+        main=requests.get('https://api.planet.com/auth/v1/experimental/public/my/subscriptions', auth=HTTPBasicAuth(data["key"], ''))
+        if main.status_code==200:
+            content=main.json()
+            n=(str(main.json()).count("quota_sqkm"))
+            i=0
+            while i<n:
```

Comment:
No need to set loop counters like this in Python; you can delete L120-L121 & L135; then just change L122 to `for item_id in content`

Reply:
Resolved